2025-08-07 08:16:47 GMT [22]: [3-1] user=,db=,app=,client= LOG:  starting PostgreSQL 17.5 (Debian 17.5-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
2025-08-07 08:16:47 GMT [22]: [4-1] user=,db=,app=,client= LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-08-07 08:16:47 GMT [22]: [5-1] user=,db=,app=,client= LOG:  listening on IPv6 address "::", port 5432
2025-08-07 08:16:47 GMT [22]: [6-1] user=,db=,app=,client= LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-08-07 08:16:47 GMT [27]: [1-1] user=,db=,app=,client= LOG:  database system was shut down at 2025-08-07 08:16:46 GMT
2025-08-07 08:16:47 GMT [22]: [7-1] user=,db=,app=,client= LOG:  database system is ready to accept connections
2025-08-07 08:16:57 GMT [33]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:16:57 GMT [33]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:16:57 GMT [33]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 08:16:58 GMT [33]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.231 user=postgres database=postgres host=[local]
2025-08-07 08:16:58 GMT [35]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:16:58 GMT [35]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:16:58 GMT [35]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 08:16:58 GMT [35]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.008 user=postgres database=postgres host=[local]
2025-08-07 08:16:58 GMT [37]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:16:58 GMT [37]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:16:58 GMT [37]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 08:16:58 GMT [37]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.006 user=postgres database=postgres host=[local]
2025-08-07 08:16:58 GMT [39]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:16:58 GMT [39]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:16:58 GMT [39]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 08:16:58 GMT [39]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.003 user=postgres database=postgres host=[local]
2025-08-07 08:17:16 GMT [166]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:17:16 GMT [166]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:17:16 GMT [166]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-07 08:17:16 GMT [166]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-07 08:17:32 GMT [167]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection received: host=172.20.0.11 port=57402
2025-08-07 08:17:32 GMT [167]: [2-1] user=replicator,db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection authenticated: identity="replicator" method=md5 (/var/lib/postgresql/data/pg_hba.conf:18)
2025-08-07 08:17:32 GMT [167]: [3-1] user=replicator,db=[unknown],app=[unknown],client=172.20.0.11 LOG:  replication connection authorized: user=replicator application_name=pg_basebackup
2025-08-07 08:17:32 GMT [25]: [1-1] user=,db=,app=,client= LOG:  checkpoint starting: force wait
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 08:17:32 GMT [30]: [1-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 08:17:32 GMT [30]: [2-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 08:17:33 GMT [30]: [3-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 08:17:33 GMT [30]: [4-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 08:17:34 GMT [30]: [5-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 08:17:34 GMT [30]: [6-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
2025-08-07 08:17:34 GMT [30]: [7-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "000000010000000000000001" failed too many times, will try again later
2025-08-07 08:17:46 GMT [181]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:17:46 GMT [181]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:17:46 GMT [181]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-07 08:17:46 GMT [181]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-07 08:18:16 GMT [190]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 08:18:16 GMT [190]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 08:18:16 GMT [190]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-07 08:18:16 GMT [190]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
